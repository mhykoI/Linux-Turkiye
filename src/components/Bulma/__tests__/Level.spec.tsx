import React from 'react';
import renderer from 'react-test-renderer';
import Level from '../Level';

describe('<Level />', () => {
  test('should render correctly', () => {
    const tree = renderer
      .create(
        <Level
          left={{
            items: [<h1 key="hi">Hi There!</h1>],
          }}
          right={{
            items: [<span key="hey">Hey, again!</span>],
            textCentered: true,
          }}
        />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
