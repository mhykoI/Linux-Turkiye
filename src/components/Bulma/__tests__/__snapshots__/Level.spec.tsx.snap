// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Level /> should render correctly 1`] = `
<nav
  className="level"
>
  <div
    className="level-left"
  >
    <div
      className="level-item"
    >
      <h1>
        Hi There!
      </h1>
    </div>
  </div>
  <div
    className="level-right"
  >
    <div
      className="level-item has-text-centered"
    >
      <span>
        Hey, again!
      </span>
    </div>
  </div>
</nav>
`;
